capstone/src/test/javascript/calendarTest.js

```diff
@@ -0,0 +1,21 @@
+describe('', function() {
```

Comment:
same comment here - I don't think this test is actually leveraging your javascript under test. can you move functions to be global in calendar.js and fix up the test?

Reply:
Ok.

Reply:
Wait actually if I still want to use dynamic loading of the Google Charts API, then it must be inside the callback function.